tests/conftest.py

```diff
@@ -56,3 +56,14 @@ async def tickit_task(request):
         await t
     except asyncio.CancelledError:
         pass
+
+
+@pytest.fixture
+def event_loop():
+    """Manage instance of event loop for runner test cases."""
+    try:
+        loop = asyncio.get_running_loop()
+    except RuntimeError:
+        loop = asyncio.new_event_loop()
+    yield loop
+    loop.close()
```

Comment:
Not sure I understand in what way this is different to what `pytest-asyncio` does, is there a reason we can't make the tests that use these `async def` with `@pytest.mark.asyncio`?